CHANGELOG/CHANGELOG-v0.5.0.md

```diff
@@ -0,0 +1,8 @@
+# Flyte V0.5.0
+
+## Infrastructure
+
+## Core Platform
+
```

Comment:
- [Run to completion](https://flyte.readthedocs.io/en/latest/user/features/on_failure_policy.html) to enable workflows to continue executing even if one or more branches fail.